### PR TITLE
Fix bug of Div

### PR DESCRIPTION
**Pull request by nomaddo**
Fix bug of division.
When we have`Div(a, b)` and 

- a is not constant
- b is constant

then the test case will failed.

----

Ci failed due to

- black (due to pysen)
- onnx

I append the version restriction)

### CODE REVIEW

**Comment by nomaddo** [comment]
Version up of `onnxruntime` may break our tests...

File: .circleci/config.yml [hunk 1]
```diff
@@ -149,11 +149,11 @@ jobs:
       - run:
           name: Install python test tools
           command: |
-            pip3 install pytest pysen isort black
+            pip3 install pytest pysen isort black==20.8b1
```
Comment (ishiy1993) [inline .circleci/config.yml:152]: ちなみにblackのバージョンを固定しているのはなぜですか?

**Review by ishiy1993 — APPROVED** [review]
LGTM

File: .circleci/config.yml (hunk 1 above)
Comment (nomaddo) [reply .circleci/config.yml:152]: Due to pysen.  pysen supports limited version of black.

**Review by notogawa — APPROVED** [review]
LGTM

**Comment by nomaddo** [comment]
🙏

**Pull request merged**